parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -295,6 +295,10 @@ function extractKubectl(){
 function ensureJournal(){
     systemctl daemon-reload
     systemctlEnableAndCheck systemd-journald.service
+    echo "Storage=persistent" >> /etc/systemd/journald.conf
+    echo "SystemMaxFileSize=10M" >> /etc/systemd/journald.conf 
```

Comment:
@feiskyer are these the default values for `SystemMaxFileSize`, `SystemMaxFiles`, and `MaxFileSec`? Here is a representative example of the current `/etc/systemd/journald.conf` config on a deployed cluster:

```
[Journal]
#Storage=auto
#Compress=yes
#Seal=yes
#SplitMode=uid
#SyncIntervalSec=5m
#RateLimitInterval=30s
#RateLimitBurst=1000
#SystemMaxUse=
#SystemKeepFree=
#SystemMaxFileSize=
#SystemMaxFiles=100
#RuntimeMaxUse=
#RuntimeKeepFree=
#RuntimeMaxFileSize=
#RuntimeMaxFiles=100
#MaxRetentionSec=
#MaxFileSec=1month
#ForwardToSyslog=yes
#ForwardToKMsg=no
#ForwardToConsole=no
#ForwardToWall=yes
#TTYPath=/dev/console
#MaxLevelStore=debug
#MaxLevelSyslog=debug
#MaxLevelKMsg=notice
#MaxLevelConsole=info
#MaxLevelWall=emerg
```

@slack @brendanburns @khenidak any thoughts on what retention/rotation settings we should deliver as static config with a change to persisting journald logs? I'm O.K. with making this static config btw, but open to thoughts that we should do the work to make these low-level knobs user-configurable.

Reply:
Spent some time trying to understand the rat's nest of options...

I'd focus on high-water mark for disk usage as that is most likely to impact customers:
* `SystemMaxUse` == 1GB
* `RuntimeMaxUse` == 1GB

If unset, both `MaxUse` values would be 10% of the disk. Paired with acs-engine defaults that'd land us at 3GB each.

Most of the other params default to okay-ish values:
* Max file size == 1/8th MaxUse
* Max file sec == 1 month

With this change we should probably turn off `ForwardToSyslog`.

Reply:
Good suggestions @slack, thanks!

@feiskyer: let's set the following config options:

SystemMaxUse=1G
RuntimeMaxUse=1G
ForwardToSyslog=no

(@feiskyer, kindly verify the gigabyte/gibibyte unit character is `G`. According to my cursory research it is: http://man7.org/linux/man-pages/man5/journald.conf.5.html)

I'll revisit this after the holiday break, thanks again @feiskyer!

Reply:
@jackfrancis thanks, updated.